src/ui/details-bottle.ui

```diff
@@ -102,25 +102,32 @@
         </child>
     </object>
     <object class="GtkPopover" id="pop_run">
-        <style>
-            <class name="menu"/>
-        </style>
         <child>
             <object class="GtkBox">
                 <property name="orientation">vertical</property>
+                <property name="valign">center</property>
+                <property name="halign">center</property>
+                <property name="hexpand">True</property>
+                <property name="vexpand">True</property>
                 <property name="margin-top">6</property>
                 <property name="margin-bottom">6</property>
                 <property name="margin-start">6</property>
                 <property name="margin-end">6</property>
                 <child>
                     <object class="GtkModelButton" id="btn_run_args">
+                        <property name="valign">center</property>
+                        <property name="halign">center</property>
+                        <property name="margin-bottom">5</property>
                         <property name="text" translatable="yes">Run with Arguments…</property>
                     </object>
                 </child>
                 <child>
                     <object class="GtkCheckButton" id="check_terminal">
                         <property name="label" translatable="yes">Launch in Terminal…</property>
                         <property name="halign">start</property>
+                        <property name="margin-bottom">5</property>
+                        <property name="use-underline">True</property>
+                        <property name="use_underline">True</property>
```

Comment:
I think this line is a mistake

Reply:
Yeah it is, I've removed it in the latest commit